packages/web-app-files/src/store/mutations.js

```diff
@@ -91,21 +91,29 @@ export default {
   SET_CLIPBOARD_ACTION(state, action) {
     state.clipboardAction = action
   },
+  SET_LATEST_SELECTED_FILE(state, file) {
```

Comment:
I'd rather call this `SET_LATEST_SELECTED_FILE_ID` to make clear the method does not expect a resource object here.

```suggestion
  SET_LATEST_SELECTED_FILE_ID(state, fileId) {
```